paderbox/array/interval/core.py

```diff
@@ -195,7 +195,7 @@ class ArrayInterval:
     from_str = staticmethod(ArrayInterval_from_str)
     inverse_mode = False
 
-    def __init__(self, array, inverse_mode: bool = False):
+    def __init__(self, array, *ignore, inverse_mode: bool = False):
```

Comment:
`def __init__(self, array, *, inverse_mode: bool = False):` does exactly what you want without the `TypeError` below.

Reply:
So we know, e.g. the parameter we gave has no purpose?

Reply:
Ok, I understand, i didn't know it worked like this